jsonc/__init__.py

```diff
@@ -60,6 +81,38 @@ def _remove_trailing_comma(text: str) -> str:
     )
 
 
+def _add_trailing_comma(text: str) -> str:
+    return re.sub(
+        _ADD_TRAILING_COMMA,
+        lambda x: x.group(1) or x.group(2) + ",",
+        text,
+        flags=re.DOTALL | re.VERBOSE,
+    )
+
+
+def _make_comment(text: str, indent=0) -> str:
+    return "\n".join(
+        " " * indent + "// " + line if line else "" for line in text.splitlines()
+    )
+
+
+def _get_comments(comments, key):
+    if isinstance(comments, dict):
+        if isinstance(key, str):
+            key = json.loads(key)
+        comments = comments.get(key)
```

Comment:
may be None

Reply:
It's intentional, not all elements have to have comments